js/__loader.js

```diff
@@ -283,6 +288,22 @@
 
   global.process = loader.require('process');
   global.Buffer = loader.require('buffer').Buffer;
+  const stream = loader.require('stream');
+  class StdoutStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.write(String(chunk));
+      callback();
+    }
+  }
+  class StderrStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.writeError(String(chunk));
+      callback();
+    }
+  }
```

Comment:
You mean with the callbacks? That's because that's the way the internal API is made in Node (plus, there's really no way to Promise that). It's fine.